src/styles/blocks/card.css

```diff
@@ -0,0 +1,60 @@
+.card {
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  width: 166px;
+  height: 356px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__computerImage {
+  width: 160px;
+  height: 134px;
+  padding: 0 3px 40px;
+}
+
+.card__title {
+  margin: 0;
+  width: 166px;
+  height: 36px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  padding-bottom: 4px;
+}
+
+.card__code {
+  font-style: normal;
+  margin: 0;
```

Comment:
remove margin from all p tags at the beginning of the file
```suggestion
```

Reply:
also I don't know why when I delete this, default margins appears. I'll commit for demonstration now